LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2014-2017 Nikolay Novik
+Copyright (c) 2017-2020 @aohan237
```

Comment:
@aohan237 I took your github nickname, feel free to post a suggestion with another name.

Reply:
this is fine, thanks